apps/nuxt3-ssr/components/CohortCard.vue

```diff
@@ -8,6 +9,7 @@ const props = withDefaults(
     cohort: ICohort;
     schema: string;
     compact?: boolean;
+    catalogue?: string;
```

Comment:
name ? id ? ...

Reply:
currently it is a network id. However I intend to generalize. We also call it 'schema' so I suppose we don't specify 'id' here?

Reply:
the Cohort card does not need to know about the catalogue, it only needs an link / to / url